index.php

```diff
@@ -5,7 +5,6 @@
 include("db.php");     // include txtDB
 include("conf.php");   // settings
 include("header.inc.php");
-include("addons.php");
```

Comment:
I removed the redundant addons file, the standard php package may work but it's best to include a tip that if the application fails to start it's recommended that the php-ldap package is installed.